lib/ui/drawing_space/tools_bottom_sheet.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+import 'package:paintroid/tool/tool.dart';
+import 'package:paintroid/ui/shared/icon_button_with_label.dart';
+
+class ToolsBottomSheet extends StatelessWidget {
+  const ToolsBottomSheet({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    const tools = ToolData.allToolsData;
+    return Column(
```

Comment:
Did you try this?
https://api.flutter.dev/flutter/widgets/GridView/GridView.count.html

Reply:
Refactored to Gridview.count, better for readability. Thanks